images/nginx/build.sh

```diff
@@ -55,41 +56,6 @@ if [[ ${ARCH} == "ppc64le" ]]; then
   clean-install software-properties-common
 fi
 
-apt-get update && apt-get dist-upgrade -y
-
-# install required packages to build
-clean-install \
-  bash \
-  build-essential \
-  curl ca-certificates \
-  libgeoip1 \
-  libgeoip-dev \
-  patch \
-  libpcre3 \
-  libpcre3-dev \
-  libssl-dev \
-  zlib1g \
-  zlib1g-dev \
-  libaio1 \
-  libaio-dev \
-  openssl \
-  libperl-dev \
-  cmake \
-  util-linux \
-  lua5.1 liblua5.1-0 liblua5.1-dev \
```

Comment:
As we are using Luajit now these are not needed. I moved to the rest of the packages to Dockerfile so that it is cached.

Reply:
Please don't remove this because luajit is not available in ppc64le and s390x